### PR TITLE
Task dev 1

### PR DESCRIPTION
**Pull request by VladimirVolkovD**

### CODE REVIEW

File: DEV-1/ArgsToString.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Text;
+
+namespace Dev1
+{
```
Comment (IgorKheidorov) [inline DEV-1/ArgsToString.cs]: Comment according to code style

File: DEV-1/ArgsToString.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    static class ArgsToString
+    {
+        /// <summary>
+        /// The ArgsToString class that forms a string from the command line arguments
+        /// </summary>
+        public static string ArrayToString(this string[] args)
+        {
```
Comment (IgorKheidorov) [inline DEV-1/ArgsToString.cs]: Good

File: DEV-1/ArgsToString.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Text;
+
+namespace Dev1
+{
+    static class ArgsToString
+    {
+        /// <summary>
+        /// The ArgsToString class that forms a string from the command line arguments
+        /// </summary>
```
Comment (IgorKheidorov) [inline DEV-1/ArgsToString.cs]: Not necessary class in argument is in ""

File: DEV-1/Program.cs [hunk 4]
```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Linq;
```
Comment (IgorKheidorov) [inline DEV-1/Program.cs:2]: Not used using

File: DEV-1/Program.cs [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+	/// MaximuNumberOfUnequalConsecutive
+	/// searches maximum number of unequal symbols in the string
+	/// and searches some exceptions
+	/// </summary>
+	class MaximuNumberOfUnequalConsecutive
+	{
+		static void Main(string[] args)
+		{
+			if (args.Length != 0)// Checking the condition for the argument
+			{
```
Comment (IgorKheidorov) [inline DEV-1/Program.cs]: Bad name for variable, better maximumUniqueCharsCount

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: DEV-1/Program.cs (hunk 4 above)
Comment (VladimirVolkovVD) [reply DEV-1/Program.cs:2]: we need System.Linq; for convert from 'char' to 'string'	 in line 40

**Review by IgorKheidorov — APPROVED** [review]

**Pull request merged**